turbinia/task_manager.py

```diff
@@ -254,6 +255,14 @@ def add_evidence(self, evidence_):
         job_count += 1
         turbinia_jobs_total.inc()
 
+    try:
+      evidence_.validate_attributes()
+    except TurbiniaException as exception:
+      log.error(f'Unsuccessful in writing new evidence in redis: {exception}')
```

Comment:
"'Unsuccessful in writing new evidence in redis" -> 'Error writing new evidence to redis"